nextflow.config

```diff
@@ -13,19 +13,16 @@ params {
     input                       = null
 
     // Main options
-    fasta                       = null
```

Comment:
I guess since you dont use this param here, you don't need the function in the main.nf